myapp/settings.py

```diff
@@ -161,11 +161,12 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.2/howto/static-files/
 
-STATIC_ROOT = os.path.join(BASE_DIR, "myapp", "staticfiles")
+STATIC_ROOT = os.path.join(BASE_DIR, "myapp", "staticfiles") if not DEBUG else os.path.join(BASE_DIR, "myapp", "static")
```

Comment:
This should always be staticfiles right? This is when you run manage.py collectstatic. When you run npm run dev, it shouldnt look here I believe